sql/updates/world/3.3.5/2021_80_98_12_world.sql

```diff
@@ -0,0 +1,386 @@
+DELETE FROM `creature_summon_groups` WHERE `summonerId`=33186; -- Razorscale
+INSERT INTO `creature_summon_groups` (`summonerId`,`summonerType`,`groupId`,`entry`,`position_x`,`position_y`,`position_z`,`orientation`,`summonType`,`summonTime`) VALUES
+(33186,0,1,33210,585.6672,-104.4477,391.6004,1.518436,8,0), -- Expedition Commander
+(33186,0,1,33287,592.5033,-98.55198,391.6004,5.742133,8,0), -- Expedition Engineer
+(33186,0,1,33287,589.5328,-95.32281,391.6004,5.51524,8,0 ), -- Expedition Engineer
+(33186,0,1,33287,594.3019,-94.38184,391.6004,4.817109,8,0), -- Expedition Engineer
+(33186,0,1,33816,600.7484,-104.8482,391.6082,4.852015,8,0), -- Expedition Defender
+(33186,0,1,33816,596.3798,-110.2639,391.6004,4.852015,8,0), -- Expedition Defender
+(33186,0,1,33816,576.5787,-113.1111,391.6004,4.29351,8,0 ), -- Expedition Defender
+(33186,0,1,33816,570.4106,-108.7936,391.6004,4.13643,8,0 ), -- Expedition Defender
+(33186,0,1,33816,588.7609,-114.8663,391.6004,4.852015,8,0), -- Expedition Defender
+(33186,0,1,33816,566.4739,-103.6336,391.6004,4.363323,8,0), -- Expedition Defender
+(33186,0,1,33259,583.61  ,-110.9356,391.6004,4.939282,8,0), -- Expedition Trapper
+(33186,0,1,33259,578.1771,-107.6289,391.6004,4.852015,8,0), -- Expedition Trapper
+(33186,0,1,33259,588.2538,-108.7151,391.6004,4.991642,8,0), -- Expedition Trapper
+(33186,0,1,33233,630.2436,-276.2591,392.3122,1.797689,8,0), -- Razorscale Controller
+(33186,0,1,33233,638.2416,-272.1735,392.1351,1.815142,8,0), -- Razorscale Controller
+(33186,0,1,33233,633.6514,-287.3748,391.8054,1.797689,8,0), -- Razorscale Controller
+(33186,0,1,33233,605.9334,-140.0912,391.6004,4.485496,8,0), -- Razorscale Controller
+(33186,0,1,33233,572.4106,-138.6564,393.9044,4.764749,8,0), -- Razorscale Controller
+(33186,0,1,33233,558.9486,-142.9874,391.6004,4.764749,8,0), -- Razorscale Controller
+(33186,0,1,33233,589.7287,-137.1148,393.9011,4.485496,8,0), -- Razorscale Controller
+-- 10 man group
+(33186,0,2,33282,589.6996,-134.6657,391.6004,4.555309,8,0), -- Razorscale Harpoon Fire State
+(33186,0,2,33282,572.0398,-136.2224,391.2637,4.642576,8,0), -- Razorscale Harpoon Fire State
+-- 25 man group
+(33186,0,3,33282,559.5352,-140.9866,391.6004,4.642576,8,0), -- Razorscale Harpoon Fire State
+(33186,0,3,33282,606.2806,-137.2628,391.6004,4.537856,8,0); -- Razorscale Harpoon Fire State
+
+DELETE FROM `gossip_menu_option` WHERE `menu_id`=10314;
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `OptionBroadcastTextID`, `option_id`, `npc_option_npcflag`, `action_menu_id`, `action_poi_id`, `box_coded`, `box_money`, `box_text`, `BoxBroadcastTextID`, `VerifiedBuild`) VALUES
+(10314, 0, 0, 'We are ready to help!', 33353, 1, 1, 0, 0, 0, 0, '', 0, 0);
+
+DELETE FROM `creature_text` WHERE `entry`=33210;
+INSERT INTO `creature_text` (`entry`,`groupid`,`id`,`text`,`type`,`language`,`probability`,`emote`,`duration`,`sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
```

Comment:
- See commit https://github.com/TrinityCore/TrinityCore/commit/d17a45ee7e348d6fc9146dd81e0457316eef1a62 (2017-05-04 `sql/updates/world/3.3.5/2017_05_04_00_world.sql`) for `creature_text` changes.
- See commit https://github.com/TrinityCore/TrinityCore/commit/5ff8601d36a14e95f9c227db1b85253ff36e551e (2017-05-04 `sql/updates/world/3.3.5/2017_05_04_02_world.sql` for `gossip_menu` and `gossip_menu_option` changes.